custom_components/mqtt_vacuum_camera/utils/camera/camera_processing.py

```
@@ -25,7 +23,9 @@
 from custom_components.mqtt_vacuum_camera.utils.files_operations import (
     async_get_active_user_language,
```

Ruff (0.8.2)

`custom_components.mqtt_vacuum_camera.utils.files_operations.async_get_active_user_language` imported but unused

Remove unused import: `custom_components.mqtt_vacuum_camera.utils.files_operations.async_get_active_user_language`

(F401)